hospitals.Rmd

```diff
@@ -44,20 +62,26 @@ addLegendCustom <- function(map, colors, labels, sizes, opacity = 0.8, stroke, t
 }
 
 map <- leaflet(data = hospitals_bogota) %>% 
-  addProviderTiles(providers$CartoDB.DarkMatter) %>% 
+  addProviderTiles(providers$CartoDB) %>%  #planB:CartoDB.DarkMatter
+  addPolygons(data = seismic_hazard , 
+              stroke = FALSE, 
+              fillOpacity = 0.5,
+              fillColor = ~pal_hazard(reclass),
+              label = ~reclass) %>% 
   addCircleMarkers(radius = ~ifelse(H_Level == "SMALL - General surgery and open 24 hrs", 4, 
                                     ifelse(H_Level == "MEDIUM -Two Specialty services", 8, 15)), 
                    color = ~pal(H_Sector), 
-                   fillOpacity = .5, 
+                   fillOpacity = 0.9, 
                    stroke = FALSE, 
-                   label = ~Hospital_name,
+                   label = ~Hospital_name, 
                    popup = paste0("Surgery Rooms:", hospitals_bogota$SurgeryRm)) %>%
   addLegendCustom(colors = c("gray", "gray", "gray"), 
                   labels = c("Small - General surgery and open 24 hrs", 
                              "Medium -Two Specialty services", 
                              "Big - High medical specialties"), 
                   sizes = c(8, 15, 25), # circles size in legend look diferent so I had to exxagerage (increase) values
-                  stroke = FALSE) %>% 
+                  stroke = FALSE) %>%
+# addLegend(pal = pal_hazard, values = ~reclass, title = "Seismic Hazard") %>% # Does not work
```

Comment:
Still need to figure out what is the problem with the legend for the hazards information. This line.